app/views/admin/users/index/controls/_export.html.erb

```diff
@@ -0,0 +1,5 @@
+<a href="<%= export_admin_users_path %>.csv" target="_blank">
```

Comment:
I didn't spend too much time on this because at the moment the icon is enclosed by a link, however in the next PR, the icon will instead behave as a button since we first show a confirmation modal.